ConfigAdmin/src/main/resources/wsdl/EntityConfigAdmin.wsdl

```diff
@@ -150,6 +177,36 @@
             </output>
         </operation>
 
+        <operation name="CreateCertificate">
+            <soap:operation soapAction="urn:CreateCertificate"/>
+            <input name="CreateCertificateRequest">
+                <soap:body use="literal"/>
+            </input>
+            <output name="SimpleCertificateResponse">
+                <soap:body use="literal"/>
+            </output>
+        </operation>
+
+        <operation name="CreateCSR">
+            <soap:operation soapAction="urn:CreateCSR"/>
+            <input name="CreateCSRRequest">
+                <soap:body use="literal"/>
+            </input>
+            <output name="SimpleCertificateResponse">
+                <soap:body use="literal"/>
+            </output>
+        </operation>
+
+        <operation name="DeleteGatewayNew">
```

Comment:
why "DeleteGatewayNew"?

Reply:
if for the cancel-button clean up the gateway_new.jks